### ISSUE TITLE
Issue with package on marketplace

### ISSUE THREAD

**Issue by chase-burandt** [umbraco/Umbraco.Marketplace.Issues#56]
This package has been imported into the marketplace previously https://www.nuget.org/packages/Our.Umbraco.Blend.Sitemap/

I can search for the package (https://marketplace.umbraco.com/search/blend)  and when I click on Blend Sitemap I get a 404.

When I try to validate (Our.Umbraco.Blend.Sitemap) the package, there is a CORS error that occurs.

**Comment by nathanwoulfe** [comment]
Hey @chase-burandt, thanks for letting us know. Not immediately sure what is happening here, but will look into it and get it resolved.

**Comment by AndyButland** [comment]
This was an interesting one... I got a few more grey hairs now after trying to figure out what was going on here.  It stems from your package name happening to end with `.Sitemap`.  This translates in the website to an API call to `GET /api/v1/packages/our.umbraco.blend.sitemap` and it seems IIS/Azure Web Apps considers this a request to a file with a `.sitemap` extension.  Which [by default is blocked](https://stackoverflow.com/questions/12828476/what-file-extensions-are-blocked-by-default-in-iis).

We've found some configuration that allows this request, so the issue will be resolved when we next deploy an update to the website, which will happen a bit later this week.
